### PR TITLE
chore: update README about docker and cozy.tools :notebook:

### PR DESCRIPTION
**Pull request by nono**
:note

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,18 +63,12 @@ $ yarn watch:browser
 
 ```sh
 # in another terminal, run the docker container
-$ docker run --rm -it -p 8080:8080 -v "$(pwd)/build":/data/cozy-app cozy/cozy-app-dev
+$ docker run --rm -it -p 8080:8080 -v "$(pwd)/build":/data/cozy-app/drive cozy/cozy-app-dev
```
Comment (kosssi) [inline README.md]: can you replace by `$ yarn stack:docker`, please ?

Comment (nono) [inline README.md]: Done

**Comment by nono** [comment]
I've updated the PR

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -50,15 +50,15 @@ Fork the project on GitHub and [check out your copy locally][forking].
 ```
 $ git clone github.com/cozy/cozy-drive.git
 $ cd cozy-drive
-$ git remote add fork git://github.com/yourusername/cozy-drive.git
+$ git remote add fork git://github.com/cozy/cozy-drive.git
```
Comment (enguerran) [inline CONTRIBUTING.md]: If the remote repository is cozy's, then contributors won't be able to `git push` on [step 7](https://github.com/cozy/cozy-drive/pull/257/files#diff-6a3371457528722a734f3c51d9238c13R93)

Comment (nono) [inline CONTRIBUTING.md]: Good catch

**Pull request merged**